src/test/scala/org/ergoplatform/serialization/SerializationTests.scala

```diff
@@ -34,6 +42,7 @@ class SerializationTests extends ErgoPropertyTest with scorex.testkit.Serializat
 
   property("ErgoTransactionGen serialization") {
     checkSerializationRoundtrip(invalidErgoTransactionGen, ErgoTransactionSerializer)
```

Comment:
let's add `checkSize` inside `checkSerializationRoundtrip` and use it everywhere

Reply:
@catena2w checkSerializationRoundtrip is in scorex, and scorex doesn't have size field.